compiam/conf/raga/carnatic.yaml

```diff
@@ -1,35 +1,295 @@
+Melakarta Ragam 
+
+1	Kanakaangi:
```

Comment:
I would add these numbers as fields within the raga...

Ratnaangi:
          arohana: ['S', 'R₁', 'G₁', 'M₁', 'P', 'D₁', 'N₂', 'Ṡ'] 
          avarohana: ['S', 'N2', 'D1', 'P', 'M1', 'G1', 'R1', 'S'] 
          number: 2